cpp-client/README.md

```diff
@@ -108,14 +108,27 @@ on them anymore so we do notguarantee they are current for those platforms.
    cmake \
        -DCMAKE_INSTALL_LIBDIR=lib \
        -DCMAKE_CXX_STANDARD=17 \
-       -DCMAKE_INSTALL_PREFIX=${DHCPP}/local \
+       -DCMAKE_INSTALL_PREFIX=${DHCPP} \
        -DCMAKE_BUILD_TYPE=RelWithDebInfo \
        -DBUILD_SHARED_LIBS=ON \
        .. \
      && \
        make -j$NCPUS install
    ```
 
+   If you need `make` to generate detailed output of the commands it is running
+   (which may be helpful for debugging the paths being used etc),
+   you can set the environment variable `VERBOSE=1`.
+   This is true in general of cmake-generated Makefiles.
+   It is useful to keep the output of make for later reference
+   in case you need to be sure of the exact compiler flags
+   that were used to compile the library and its objects.
+   You can tweak the last command above as
+
+   ```
+   VERBOSE=1 make -j$NCPUS install 2>&1 | tee make-install.log
```

Comment:
you can also say `make VERBOSE=1`...  maybe that's simpler or it doesn't matter.

Reply:
I feel a bit icky about `make VERBOSE=1 -j32` because of the order of definition versus flag, but it happens to work, at least in GNU make, and in practice all these linux distributiosn are GNU.
I believe I prefer the `environment variable for the process` regular shell syntax tho, because that's more general.
Do you care if we just leave it as is?

Update: Corey and myself discussed, we are keeping it as is for now.